sdk/storage/storage-blob/test/perfstress/track-1/download.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  Aborter,
+  BlobURL,
+  BlockBlobURL,
+  ContainerURL,
+  ServiceURL,
+  SharedKeyCredential,
+  StorageURL
+} from "@azure/storage-blob";
+import { PerfStressOptionDictionary, PerfStressTest } from "@azure/test-utils-perfstress";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config({ path: "../../../.env" });
+
+interface StorageBlobDownloadTestOptions {
+  size: number;
+}
+
+const account = process.env.ACCOUNT_NAME || "";
+const accountKey = process.env.ACCOUNT_KEY || "";
+
+const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
+
+const blobServiceClient = new ServiceURL(
+  // When using AnonymousCredential, following url should include a valid SAS or support public access
+  `https://${account}.blob.core.windows.net`,
+  StorageURL.newPipeline(sharedKeyCredential)
+);
+const containerName = `newcontainer${new Date().getTime()}`;
+const blobName = `newblob${new Date().getTime()}`;
+const containerClient = ContainerURL.fromServiceURL(blobServiceClient, containerName);
+const blobClient = BlobURL.fromContainerURL(containerClient, blobName);
+const blockBlobClient = BlockBlobURL.fromBlobURL(blobClient);
+
+export class StorageBlobDownloadTest extends PerfStressTest<StorageBlobDownloadTestOptions> {
+  public options: PerfStressOptionDictionary<StorageBlobDownloadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 10
+    }
+  };
+
+  public async globalSetup() {
+    const createContainerResponse = await containerClient.create(Aborter.none);
+    console.log(
+      `Create container ${containerName} successfully`,
+      createContainerResponse.requestId
+    );
+
+    // Create a blob
+    const uploadBlobResponse = await blockBlobClient.upload(
+      Aborter.none,
+      Buffer.alloc(this.parsedOptions.size.value!),
+      this.parsedOptions.size.value!
+    );
+    console.log(`Uploaded block blob ${blobName} successfully`, uploadBlobResponse.requestId);
+  }
+
+  public async globalCleanup() {
```

Comment:
I recommend creating a base class containing the logic common to all the storage tests, like creating and deleting the container.  .NET uses the following base classes:

https://github.com/mikeharder/azure-sdk-perf/tree/master/net/Azure.Storage.Blobs.PerfStress/Core

Reply:
Makes sense, will do!

Reply:
Added